src/pages/en/reference/integrations-reference.md

```diff
@@ -191,10 +191,10 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
 
 **Type:** [`ViteDevServer`](https://vitejs.dev/guide/api-javascript.html#vitedevserver)
 
-A mutable instance of the Vite server used in "dev" and "preview" mode. For instance, this is [used by our Partytown integration](https://github.com/withastro/astro/tree/main/packages/integrations/partytown) to inject the Partytown server as middleware:
+A mutable instance of the Vite server used in "dev" and "preview" mode. For instance, this is [used by our Partytown integration](/en/guides/integrations-guide/partytown/) to inject the Partytown server as middleware:
 
 ```js
-import
+import 
```

Comment:
Looks like this snuck in for some reason, but otherwise LGTM! Congrats on wrangling the hippo-check 🥳 

```suggestion
import
```